spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaSettings.java

```diff
@@ -61,6 +61,9 @@
  *     mime-types: text/*, application/json
  *     excluded-user-agents: some-user-agent, another-user-agent
  *     min-response-size: 1KB
+ *   security:
+ *     enabled: true
+ *     ports: 8080
```

Comment:
Question: Is it possible to use `management.server.port` instead?

Reply:
It is not possible right now. However, if `armeria.security.enabled` is true, and ` armeria.security.ports` is not defined, I think it's a good approach to look at `management.server.port` and assign them.

Reply:
Question: Would you like to remove the `armeria.security.ports` value instead of the optional value?

Reply:
I tried `management.server.port` and it doesn't bind to that value. Do you want this property to be an additional binding?

Reply:
> Question: Do you want to remove the `armeria.security.ports` value instead of the optional value?

Yes, we might want to add it later but not until there's a user request.

> I tried `management.server.port` and it doesn't bind to that value. Do you want this property to be an additional binding?

Yes, please.

Reply:
Thanks for the reply. :smile:

We can apply security by checking `management.server.port` when using actuator. However, we're obscure which ports to secure `internal/docs`, `internal/healthcheck`, and `internal/metrics`.  :flushed:

Reply:
That's a good question. Perhaps we could default to all services under `/internal/` and make it configurable?

Reply:
How about removing `armeria.security.ports` and adding `armeria.internal.port`?

Reply:
I've configured the default port(8001) if `armeria.security.enabled` is true, and have it protect all services under `/internal/` on other ports.

Reply:
Question: I specified the default port, but when the user specified `management.server.port` I didn't remove the default port. Should I avoid specifying a default port?